WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -354,11 +351,6 @@ protected void onCreate(Bundle savedInstanceState) {
             mSite = (SiteModel) savedInstanceState.getSerializable(WordPress.SITE);
             mFullScreenDialogFragment = (FullScreenDialogFragment)
```

Comment:
The `mFullScreenDialogFragment` variable and all references to it can be removed from the `EditPostActivity` class.  It's only used in the `handleBackPressed()` method now, which was used to dismiss the `HistoryDetailFullScreenDialogFragment` and that no longer exists.